IPR Policy.md

```diff
@@ -96,7 +96,7 @@ If the [Living Standard] is depublished by the WHATWG, then no new licenses need
 
 ### 3.1. Contributors and Workstream Participants
 
-Commencing 2018-01-11, the [IPR Policy] becomes effective and all [Contributors][contributor] and [Workstream Participants] must execute the [Contributor and Workstream Participant Agreement][cw-agreement] in order to contribute or participate in [Workstreams][Workstream]. To help ensure that appropriate licensing commitments are secured from all [Contributors][contributor], individuals may [Contribute][contribution] independently to a [Living Standard] only if they do not [work in the field of Web technologies] as an employee, contractor, or agent of another person or legal entity (as set forth in the [Contributor and Workstream Participant Agreement][cw-agreement]).
+Commencing 2018-01-11, the [IPR Policy] becomes effective and all [Contributors][contributor] and [Workstream Participants] must execute the [Contributor and Workstream Participant Agreement][cw-agreement] in order to contribute or participate in [Workstreams][Workstream]. To help ensure that appropriate licensing commitments are secured from all [Contributors][contributor], [Individuals][Individual] who have executed the [Contributor and Workstream Participant Agreement][cw-agreement] may [Contribute][contribution] independently to a [Living Standard] only if they (a) do not [work in the field of Web technologies] as an employee, contractor, or agent of another person or legal entity; or (b) have been invited by the [Steering Group] to participate in one or more WHATWG [Workstreams][Workstream] (in both cases, as set forth in the [Contributor and Workstream Participant Agreement][cw-agreement]). To the extent that such [Individuals][Individual] make [Contributions][contribution], each [Individual] hereby represents and warrants that such [Individual] has all rights necessary in and to any [Contributions][contribution] it makes.
```

Comment:
I would have expected "they make" at the end here.

Reply:
```suggestion
Commencing 2018-01-11, the [IPR Policy] becomes effective and all [Contributors][contributor] and [Workstream Participants] must execute the [Contributor and Workstream Participant Agreement][cw-agreement] in order to contribute or participate in [Workstreams][Workstream]. To help ensure that appropriate licensing commitments are secured from all [Contributors][contributor], [Individuals][Individual] who have executed the [Contributor and Workstream Participant Agreement][cw-agreement] may [Contribute][contribution] independently to a [Living Standard] only if they (a) do not [work in the field of Web technologies] as an employee, contractor, or agent of another person or legal entity; or (b) have been invited by the [Steering Group] to participate in one or more WHATWG [Workstreams][Workstream] (in both cases, as set forth in the [Contributor and Workstream Participant Agreement][cw-agreement]). To the extent that such [Individuals][Individual] make [Contributions][contribution], each [Individual] hereby represents and warrants that such [Individual] has all rights necessary in and to any [Contributions][contribution] they make.
```

I agree that reads much better, referring to people as "it" is unusual, so I support making that change.